src/server/scripts/Spells/spell_priest.cpp

```diff
@@ -1174,6 +1180,189 @@ class spell_pri_empowered_renew : public AuraScript
+        if (!caster)
+            return;
+
+        caster->CastSpell(caster, SPELL_PRIEST_ENTROPIC_RIFT_AURA);
+
+        if (creatingSpell)
+        {
+            if (ObjectGuid const* targetGUID = std::any_cast<ObjectGuid>(&creatingSpell->m_customArg))
+                _targetGuid = *targetGUID;
+            _searchRadius = creatingSpell->GetSpellInfo()->GetMaxRange();
+        }
+    }
+
+    void OnUpdate(uint32 diff) override
+    {
+        _scheduler.Update(diff);
+
+        _damageTimer -= Milliseconds(diff);
+        while (_damageTimer <= 0s)
+        {
+            DealDamage();
+            _damageTimer += damagePeriod;
+        }
+    }
+
+    void UpdateMovement()
+    {
+        Unit* caster = at->GetCaster();
+        if (!caster)
+            return;
+
+        Unit* target = nullptr;
+        if (!_targetGuid.IsEmpty())
+            target = ObjectAccessor::GetUnit(*at, _targetGuid);
+
+        if (!target || !target->IsAlive())
+        {
+            target = FindNewEnemy(caster);
+            if (!target)
+            {
+                _targetGuid = ObjectGuid::Empty;
+                return;
+            }
+
+            _targetGuid = target->GetGUID();
+        }
+
+        PathGenerator path(at);
+        path.CalculatePath(target->GetPositionX(), target->GetPositionY(), target->GetPositionZ(), false);
+        at->InitSplines(path.GetPath());
+    }
+
+    void DealDamage()
+    {
+        Unit* caster = at->GetCaster();
+        if (!caster)
+            return;
+
+        float radius = at->GetMaxSearchRadius();
+        std::list<WorldObject*> targets;
+        Trinity::AnyUnfriendlyUnitInObjectRangeCheck checker(at, caster, radius);
+        Trinity::UnitListSearcher<Trinity::AnyUnfriendlyUnitInObjectRangeCheck> searcher(at, targets, checker);
+        Cell::VisitAllObjects(at, searcher, radius);
+
+        targets.remove_if([caster](WorldObject const* target)
+        {
+            return !caster->IsValidAttackTarget(target);
+        });
+
+        for (WorldObject* target : targets)
+            caster->CastSpell(target->ToUnit(), SPELL_PRIEST_ENTROPIC_RIFT_DAMAGE);
+    }
+
+    Unit* FindNewEnemy(Unit const* caster)
+    {
+        std::list<WorldObject*> targets;
+        Trinity::AnyUnfriendlyUnitInObjectRangeCheck checker(at, caster, _searchRadius);
+        Trinity::UnitListSearcher<Trinity::AnyUnfriendlyUnitInObjectRangeCheck> searcher(at, targets, checker);
+        Cell::VisitAllObjects(at, searcher, _searchRadius);
+
+        targets.remove_if([caster](WorldObject const* obj)
+        {
+            Unit const* target = obj->ToUnit();
+            return !target || !target->IsAlive() || !caster->IsValidAttackTarget(target) || !caster->IsInCombatWith(target);
+        });
+
+        if (targets.empty())
+            return nullptr;
+
+        return Trinity::Containers::SelectRandomContainerElement(targets)->ToUnit();
+    }
+
+private:
+    static constexpr Milliseconds damagePeriod = 1s;
```

Comment:
this makes the spell a static periodic which doesn't scale with haste, the AuraScript you attached to spellId 450193, however, does indeed use ATTR5_SPELL_HASTE_AFFECTS_PERIODIC, which most likely tells us it's the one doing the work at casting at enemies inside the AT.